Birthday-2024-Project/MainScenes/sapling_level_select_1.tscn

```diff
@@ -0,0 +1,108 @@
+[ext_resource type="Resource" uid="uid://c37hquqhl3grb" path="res://Resources/Levels/AgentFauna7.tres" id="4_bgm7l"]
+[ext_resource type="Resource" uid="uid://dl3mppio5oh5o" path="res://Resources/Levels/castle.tres" id="5_sv7we"]
+[ext_resource type="Resource" uid="uid://brx4plkjxoh58" path="res://Resources/Levels/cornfauner.tres" id="6_nl703"]
+[ext_resource type="Resource" uid="uid://b6gqsv185mowu" path="res://Resources/Levels/CrashsFonLevel.tres" id="7_oprcp"]
+[ext_resource type="Resource" uid="uid://b01xvdnifxq0l" path="res://Resources/Levels/DeerPat.tres" id="8_v8be4"]
+[ext_resource type="Resource" uid="uid://des2i6l5dpdk7" path="res://Resources/Levels/FaunaBirthdayCake.tres" id="9_bned0"]
+[ext_resource type="Resource" uid="uid://dhvtpu4jwgilb" path="res://Resources/Levels/FaunaFishy.tres" id="10_askip"]
+[ext_resource type="Resource" uid="uid://ci0lkd5ph0ged" path="res://Resources/Levels/Faunalevel.tres" id="11_qd3af"]
+[ext_resource type="Resource" uid="uid://dh866o0aim8pk" path="res://Resources/Levels/Fauna_is_cute.tres" id="12_fxrr1"]
+[ext_resource type="Resource" uid="uid://ce260q0n2i8ay" path="res://Resources/Levels/Fauna_Potluck.tres" id="13_tg6uo"]
+[ext_resource type="Resource" uid="uid://56yv87hnoqi7" path="res://Resources/Levels/Fauna_puzzel_level.tres" id="14_mxpps"]
+[ext_resource type="Resource" uid="uid://c8u4oy5fadvww" path="res://Resources/Levels/FaunnyBirthday.tres" id="15_i22jj"]
+[ext_resource type="Resource" uid="uid://ddl0osmlogcu8" path="res://Resources/Levels/Faunyas_Snails.tres" id="16_qvkon"]
+[ext_resource type="Resource" uid="uid://bnswhxfv3b16r" path="res://Resources/Levels/fon-level-ikarus.tres" id="17_ukcv0"]
+[ext_resource type="Resource" uid="uid://3sdoy0njw7ti" path="res://Resources/Levels/FONlevel.tres" id="18_wakqf"]
+[ext_resource type="Resource" uid="uid://mrwd3nmyd2of" path="res://Resources/Levels/FONLlama.tres" id="19_do54h"]
+[ext_resource type="Resource" uid="uid://bfi181yjqfhe8" path="res://Resources/Levels/FoN_Level.tres" id="20_ebdjn"]
+[ext_resource type="Resource" uid="uid://k1vyd3yuwkh" path="res://Resources/Levels/FON_level_hearty.tres" id="21_0gg42"]
+[ext_resource type="Resource" uid="uid://0p0xj8an8m6e" path="res://Resources/Levels/goobert.tres" id="22_agmuu"]
+
+[node name="SaplingLevelSelection" type="Control"]
+layout_mode = 3
+anchors_preset = 15
+anchor_right = 1.0
+anchor_bottom = 1.0
+grow_horizontal = 2
+grow_vertical = 2
+script = ExtResource("1_ylpox")
+
+[node name="LevelSelectMenu" parent="." node_paths=PackedStringArray("screenLogic") instance=ExtResource("2_glrcg")]
+layout_mode = 1
+screenLogic = NodePath("..")
+isCampaign = false
+isLastPage = false
+
+[node name="Title" parent="LevelSelectMenu/Panel" index="0"]
+text = "COMMUNITY"
+
+[node name="Page" parent="LevelSelectMenu/Panel" index="1"]
+visible = true
+text = "1/3"
+
+[node name="Level Button" parent="LevelSelectMenu/Panel/Level Select Buttons" index="0"]
```

Comment:
About buttons in general:
Currently, there is no in-game way to tell who is an author of each level, maybe we could utilize white areas to have names. It's currently too small so some edits would need to be done. What do you think?

Reply:
I agree, there's no distinguishing features to the level select buttons. We could probably add some text above/below or on them if we enlarge the photo image. Do we want it to say the level name, the author, or both? Both might make it too noisy, but if not both, then there's the question of where to put the other. In the level maybe?

Reply:
Could potentially show the author when hovering over a level? Like a tooltip

Reply:
I was thinking of a name only on a button and if there is a title then use it as a label at the top of the actual level

Reply:
Oh, I meant the tooltip in addition to whatever is decided on the button. So it will still say like Level Name, but if you hover over it, it says the author. Just a potential idea

Reply:
I'd prioritize submitters' name visibility over level names. So following your example: sapling name on button and level name on hover display. Not everyone named them and because it's a community celebration, showing who was here is more important than what name level has

Reply:
![image](https://github.com/Saplings-Projects/Birthday-2024/assets/9286524/1aa16a91-e396-4ea3-9517-6c902b0b54cf)

I had a new idea. I changed the level preview into two images. One is "incomplete" and one is "complete". An incomplete level shows the grid shape in a monochrome shader. A complete level shows the finished level in color. When you hover over a level, it shows the author (was previously level name, but can easily be changed).

![image](https://github.com/Saplings-Projects/Birthday-2024/assets/9286524/9fe05345-ae79-4cca-9596-fedf212fe56b)

It's more in line with how Cats Organized Neatly does it. Keeps it mostly textless. It's also nice to see the levels start filling out with color as you complete them


There's some variations in the preview image above. The new tileset is rather light, so I tried to see if it would look better darker. If you want to see them in-game, it's currently set to set level button completion randomly. At least until we get proper progress management implemented. You can back in & out of the page to see them change.

There's only the 5 level buttons under the community levels that have previews. I didn't make level images for all the levels since I wasn't sure if the tileset or levels or other things will get changed. Will probably work through them in a future update once all the submissions are in